cmake/SSGCommon.cmake

```diff
@@ -60,6 +60,17 @@ endif()
 set(SSG_HTML_GUIDE_FILE_LIST "")
 set(SSG_HTML_TABLE_FILE_LIST "")
 
+# Define validate_product FALSE for appropriate combinations
+# of products and oscap versions.
+function(define_validate_product PRODUCT)
+    set(validate_product, TRUE)
+    if ("${OSCAP_VERSION}" VERSION_LESS "1.3.4")
+        if ("${PRODUCT}" MATCHES "^(ocp4|)$")
```

Comment:
Is there a use case where `PRODUCT` is empty?

Reply:
Not really, it was an indication how to expand list of products easily.